src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PreviewWindowModel.cs

```diff
@@ -53,6 +53,19 @@ public override string ToString()
                     }
                     sb.AppendLine("");
                 }
+                if (r.NewSourceMappings?.Any() == true)
```

Comment:
https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/24181/Garbage-Collection?anchor=%E2%9C%94%EF%B8%8F-**do**-favor-%60length%60-and-%60count%60-checks-over-%60any%60

```suggestion
                if (r.NewSourceMappings?.Count > 0)
```

Reply:
👍